pkg/client/block/client_test.go

```diff
@@ -79,6 +79,23 @@ func TestBlockClient(t *testing.T) {
 				require.Equal(t, expectedHeight, lastBlock.Height())
 				require.Equal(t, expectedHash, lastBlock.Hash())
 
+				return lastBlock
+			},
+		},
+		{
+			name: "CommittedBlocksSequence successfully remaps closure",
+			fn: func() client.Block {
+				// Close the events query client to force a reconnection.
+				eventsQueryClient.Close()
```

Comment:
I think we should be closing a mocked `Connection` instead:
(see: https://github.com/pokt-network/poktroll/pull/267#discussion_r1425089102)